homework1/index.css

```diff
@@ -0,0 +1,443 @@
+        background: #59ABB0;
+        border-radius: 10px;
+        color: white;
+        font-size: 18px;
+        margin-top: 40px;
+        justify-content: center;
+        align-self: center;
+        margin: auto;
+
+    }
+
+    .top_section_image {
+        width: 100%;
+        height: auto;
+    }
+
+    .bottom_section {
+        display: flex;
+        flex-direction: column;
+        margin: 0 20px 95px 20px;
+    }
+
+    .bottom_section_content_text {
+        margin: auto;
+
+    }
+
+    .bottom_section_text {
+        padding-bottom: 18px;
+        margin: auto;
+    }
+
+    .bottom_section_button {
+        display: flex;
+        justify-content: center;
+        padding: 10px 100px;
+        background: #59ABB0;
+        border-radius: 10px;
+        text-decoration: none;
+        color: white;
+
+    }
+
+    .bottom_section_image {
+        width: 100%;
+        height: auto;
+    }
+
+    .footer_container {
+        background: #59ABB0;
+        padding: 39px 20px;
+    }
+
+    .footer_content {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    .footer_text {
+        color: white;
+        font-size: 14px;
+    }
+
+    .footer_menu {
+        display: flex;
+        flex-direction: column;
+        list-style: none;
+    }
+
+    .footer_menu_bttn {
+        margin-right: 20px;
+        font-size: 14px;
+        color: white;
+        text-decoration: none;
+    }
+}
+
+
+/*desktop*/
+
+
+
+@media (min-width:801px) {
+    body {
+        margin: 0;
+        width: 100%;
+        height: 100%;
+    }
+
+    .hamburger {
+        display: none;
+    }
+
+    .header_container {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        margin: 62px 100px 44px 100px;
+    }
+
+    .header_logo {
+        width: 145px;
+        height: 60px;
+    }
+
+
+
+    .nav_menu {
+        display: flex;
+        flex-direction: row;
+        list-style: none;
+        
+        align-items: center;
+    }
+
+    .nav_menu li {
+        margin-right: 24px;
+        text-decoration: none;
+        
+    }
+
+    .nav_menu_bttn {
+        text-decoration: none;
+        color: black;
+        
+
+        font-size: 16px;
+    }
+
+    .nav_menu_bttn:hover {
+        
+        font-weight: bolder;
+
+        transition:0.5s;
+        color: black;
+    }
+
+    .nav_menu_apply_button {
+        padding: 18px 60px;
+        background: #59ABB0;
+        border-radius: 10px;
+        font-size: 16px;
+        color: white;
+        text-decoration: none;
+        
+    }
+
+    .nav_menu_apply_button:hover {
+        background: #60DFE8;
+        transition: 0.5s;
+    }
+
+    .top_section {
+        display: flex;
+        flex-direction: row;
+        margin: 0 100px 47px 100px;
+    }
+
+    .top_section_text_content {
+        margin-right: 170px;
+        width: 705px;
+        height: 716px;
```

Comment:
Try to don't use a fixed height for the text. Please remove it.
e.g. if u need u can use min or max height